Assignment2/Document.cpp

```diff
@@ -0,0 +1,79 @@
+
+
+#include "document.h"
+
+#include<fstream>
+#include<iostream>
+#include<string>
+#include<vector>
+#include<algorithm>
+#include<map>
+#include<iterator>
+#include<iomanip>
+#include <math.h> 
+
+typedef  map<string, map<string, unsigned int >> nestedMap;
+using namespace std;
+
+//defult constructor that creates an empty document
+Document::Document()

```

Comment:
We probably don't need a default constructor at all, considering it would not be very useful

Reply:
I agree, but its on the list of things we need to have